### PR TITLE
Use select to reduce system load

### PR DESCRIPTION
**Pull request by jspricke**

### CODE REVIEW

**Comment by hifiberry** [comment]
10 seconds in way too long. With the select() improvement, I'm happy to reduce this to 1 seonds, but not 10.

File: hifiberrydsp/alsa/alsasync.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def __init__(self):
         self.dsp = Adau145x
         self.volume_register_length = self.dsp.WORD_LENGTH
         self.finished = False
-        self.pollinterval = 0.1
+        self.pollinterval = 10
```
Comment (hifiberry) [inline hifiberrydsp/alsa/alsasync.py:75]: Please change to 1 second

Comment (hifiberry) [reply hifiberrydsp/alsa/alsasync.py:75]: Alternatively add a command line option to set the interval

Comment (jspricke) [reply hifiberrydsp/alsa/alsasync.py]: 1 second (now in milliseconds) is fine with me.

**Pull request merged**